### ISSUE TITLE
Arenastats page is not updating correctly

### ISSUE THREAD

**Issue by Belodina** [chromiecraft/chromiecraft#7496]
### Bug Title

Arena Stats page incorrectly displays number of games played

### Current Behaviour

On PC and Mobile, opening the Arena Stats page displays incorrect information regarding number of games played:

![image](https://github.com/user-attachments/assets/e2c853ad-d7ec-436b-8613-db27d33d6166)

On the main page ^^ and when viewing individual player data:

![image](https://github.com/user-attachments/assets/af86c635-3564-4200-a2c2-12a8340d0681)

For example, I have lost plenty more than three games this season in 3v3 soloq.

This is the same on both Chromium based browsers I have used, Google Chrome and Brave, as well as Chrome Mobile.

### Expected Behaviour

The web page should presumably display accurate, and either live updated or updated at set times, data for all to view. As of now, it is strange to see a rank 1 soloq arena player with 0 season games.

### Steps to Reproduce

Go to https://www.chromiecraft.com/en/arena-stats/

### Browser

Other

### Device

Desktop

### Operating System

Windows 11

### URL

https://www.chromiecraft.com/en/arena-stats/

**Comment by Helias** [comment]
Hi, thanks for reporting the bug, but, can you see the correct data in game if you ask them via the 3v3soloQ NPC?

**Comment by Helias** [comment]
checking deeply, the problem seems related on how we store the season games, it's not related to the arena-stats, it shows what we have in the database.

I opened an issue [here](https://github.com/azerothcore/mod-arena-3v3-solo-queue/issues/25)  in the arena 3v3soloQ module.

Could you tell me if, you left always the arena match when it ended or if you left it even before?

Thanks for the report.

**Comment by Apathyxia** [comment]
I have provided some additional information under the [3v3soloQ module](https://github.com/azerothcore/mod-arena-3v3-solo-queue/issues/25). I was about to make a separate issue regarding it but it's so heavily related, I just commented it there. 

TLDR; Replaying games are counted on the website, and the games required for points is not accurate.

**Comment by Belodina** [comment]
Hi Helias, I typically stay until the end of the match to leave,  of course that is 100% the case for won games, sometimes leaving the game early after dying and it is a clear loss..

Speaking to the Arena 3v3 Solo NPC at 17:41 SVT, 12/11/2024:

![image](https://github.com/user-attachments/assets/0c9cb1d6-db65-45ec-9764-87e7b678cc91)

I hope this helps!

**Comment by Helias** [comment]
Speaking with @laasker we have already found the problem and the related fix, but the season games will be a bit buggy until we do not reset again the teams unfortunately

**Comment by Helias** [comment]
forgot to say, I opened a separate issue for the wrong calculation of week needed, thanks @Apathyxia for reporting it, this is an issue of the arena-stats, and it's only a "display issue", so that does not affect the in-game rules

**Comment by Helias** [comment]
It should be fixed now.

The only issue remaining about arena-stats is the wrong needed game count when the team did at least 10 games and a player already did 33% of the games.
I've reported the issue here https://github.com/azerothcore/arena-stats/issues/59, hope to fix it this month.
